### PR TITLE
Update ARG Task versions

### PR DESCRIPTION
**Pull request by vineetmimrot**
**Task name**:  ARG v2, v3

**Description**: <Describe your changes here>

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AzureResourceGroupDeploymentV2/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@types/node": "^10.17.0",
     "azure-pipelines-task-lib": "^3.0.6-preview.0",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.1",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.5",
```
Comment (bishal-pdMSFT) [inline Tasks/AzureResourceGroupDeploymentV2/package.json:7]: why is this change needed? `^` will take care of it. Or is it because of package-lock.json file?

**Review by bishal-pdMSFT — APPROVED** [review]

File: Tasks/AzureResourceGroupDeploymentV2/package.json (hunk 1 above)
Comment (vineetmimrot) [reply Tasks/AzureResourceGroupDeploymentV2/package.json:7]: ^ will take care of it, but just to be sure we will get the latest package.
